tests/scripts/arm32_ci_test.sh

```diff
@@ -38,8 +38,11 @@ function exit_if_path_absent {
     fi
 }
 
+
+# Below variables will be determined by arguments or have default value.
 __abi="arm"
 __buildConfig="Release"
+__COMPLUS_OPTIONS=" "
```

Comment:
You initialize a variable with leading underscores, but use a variable without those leading underscores.

Reply:
Thanks!